test/functional/services/dashboard/add_panel.ts

```diff
@@ -79,32 +81,42 @@ export class DashboardAddPanelService extends FtrService {
 
   async toggleFilterPopover() {
     this.log.debug('DashboardAddPanel.toggleFilter');
-    await this.testSubjects.click('savedObjectFinderFilterButton');
+    const filtersHolder = await this.find.byClassName('euiSearchBar__filtersHolder');
+    const filtersButton = await filtersHolder.findByCssSelector('button');
+    await filtersButton.click();
   }
 
   async toggleFilter(type: string) {
     this.log.debug(`DashboardAddPanel.addToFilter(${type})`);
     await this.waitForListLoading();
     await this.toggleFilterPopover();
-    await this.testSubjects.click(`savedObjectFinderFilter-${type}`);
-    await this.toggleFilterPopover();
+    const list = await this.testSubjects.find('euiSelectableList');
```

Comment:
Same with this logic that is also duplicated in `replace_panel.ts`. How about creating a page service and exposing function to "click item"?